monitors/SyntheticPegMonitor.js

```diff
@@ -83,6 +100,55 @@ class SyntheticPegMonitor {
     }
   };
 
+  // Checks difference between minimum and maximum historical price over `volatilityWindow` amount of time in the
+  // specified price feed. Fires a message if the difference exceeds the `volatilityAlertThreshold` %.
+  checkPriceVolatility = async () => {
+    // TODO: Should we allow caller to specify the Uniswap versus Medianizer pricefeed? Should we check both on each call?
```

Comment:
I think we should have two monitors for each in the future. It would be ideal to be able to specify a price feed config that you want to monitor for volatility.

Reply:
I agree. I did this by exposing two public methods that both call an internal method `_checkPricefeedVolatility(pricefeed)`